ui/src/state/chat/chat.ts

```diff
@@ -745,9 +747,8 @@ export function useMarkDmReadMutation() {
   });
 }
 
-const emptyUnreads: DMUnreads = {};
-const dmUnreadsKey = ['dm', 'unreads'];
 export function useDmUnreads() {
+  const dmUnreadsKey = ['dms', 'unreads'];
```

Comment:
we actually should move these keys up and reuse them everywhere we use the unreads key

Reply:
agreed. Rather than doing it one off here, I'll open a separate PR to extract keys more consistently.